x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/overview_logic.test.ts

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { resetContext } from 'kea';
+import { act } from 'react-dom/test-utils';
+
+import { mockKibanaContext } from '../../../__mocks__';
+
+import { mockLogicValues } from './__mocks__';
+import { OverviewLogic } from './overview_logic';
+
+let unmount: any;
+
+beforeEach(() => {
+  resetContext();
+  unmount = OverviewLogic.mount() as any;
+  jest.clearAllMocks();
+});
+
+afterEach(() => {
+  unmount();
+});
+
+describe('OverviewLogic', () => {
```

Comment:
I'd slightly prefer to keep before/after etc. within the main describe() block - any objections?

```suggestion
describe('OverviewLogic', () => {
  let unmount: any;

  beforeEach(() => {
    resetContext({});
    unmount = OverviewLogic.mount() as any;
    jest.clearAllMocks();
  });

  afterEach(() => {
    unmount();
  });

```

Reply:
Yeah that's totally fine